BrowserKit/Sources/ComponentLibrary/Buttons/PrimaryRoundedButton.swift

```diff
@@ -32,21 +32,22 @@ public class PrimaryRoundedButton: LegacyResizableButton, ThemeApplicable {
     override init(frame: CGRect) {
         super.init(frame: frame)
 
-        titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
+        configuration?.setFont(DefaultDynamicFontHelper.preferredBoldFont(
             withTextStyle: .callout,
-            size: UX.buttonFontSize)
+            size: UX.buttonFontSize
+        ))
         layer.cornerRadius = UX.buttonCornerRadius
         titleLabel?.textAlignment = .center
         titleLabel?.adjustsFontForContentSizeCategory = true
-        contentEdgeInsets = UX.contentEdgeInsets
+        configuration?.contentInsets = UX.contentInsets
```

Comment:
Same for the insets: Could you please move this to the configure method?

Reply:
Done!